BreadCrumb Control/BreadCrumb Control/BreadCrumb.swift

```diff
@@ -62,14 +70,14 @@ public class CBreadcrumbControl: UIControl{
         NotificationCenter.default.addObserver(self, selector: #selector(self.receivedUINotificationNewItems), name:NSNotification.Name(rawValue: "NotificationNewItems"), object: nil)
     }
     
+    @IBInspectable public var autoScrollEnabled = false
```

Comment:
For whatever reason in XCode (using 8.3.3), `@IBInspectable` properties don't seem to show up in the Attributes Inspector unless you explicitly define its type (i.e. `Bool`).

`@IBInspectable public var autoScrollEnabled: Bool = false`

Reply:
I defined `autoScrollEnabled` type explicitly for Interface Builder. And confirmed it on XCode 8.3.3.
![autoscrollenabled](https://user-images.githubusercontent.com/25785565/29755790-4a7598a0-8bd8-11e7-97af-1440f225c1ab.png)